### PR TITLE
Updates in the text of MTL example notebook

### PR DESCRIPTION
**Pull request by gabrielspmoreira**
This PR is just about small edits the text of the MTL example to better align with the blog post content

### CODE REVIEW

File: examples/usecases/ranking_with_multitask_learning.ipynb [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "\n",
```
Comment (rnyak) [inline examples/usecases/ranking_with_multitask_learning.ipynb:50]: <p>can we write instead "In this example notebook, we use a synthetic dataset generated from the schema of the dataset released in the <a href="https://arxiv.org/abs/2210.10629" target="_blank">TenRec paper</a>, ..." ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (rnyak) [inline examples/usecases/ranking_with_multitask_learning.ipynb:50]: <p>this example link is not correct: For more information about how the schema API works you can check this <a href="https://github.com/NVIDIA-Merlin/models/blob/main/merlin/datasets/entertainment/tenrec_video/schema.pbtxt" target="_blank">example</a>.</p><p> </p><p>I think you wanted to link to this example: https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by rnyak** [comment]
rerun tests

File: examples/usecases/ranking_with_multitask_learning.ipynb (hunk 1 above)
Comment (gabrielspmoreira) [reply examples/usecases/ranking_with_multitask_learning.ipynb:50]: <p>Thanks. Updated</p>

**Pull request merged**